modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -100,6 +100,14 @@ ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:enp1s0:none
 nameserver=4.4.4.41
 ----
 
+a|You can configure an IP address without assigning a static hostname. If a static hostname is not set by the user, it will be picked up and automatically set by a reverse DNS lookup. Use the same example sets as those above, and exclude the hostname.
```

Comment:
Just a [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#cloud-services-visual-info) note:
Last bullet about directional "up, down, left, right, etc." 

Could you say a variation of: 

> Use the same example sets as those mentioned in section a

Reply:
i like how you fixed! Makes more sense where it is now! :)